start.py

```diff
@@ -1,13 +1,43 @@
+import os
 import sys
+import __main__
 
 from PyQt5.QtWidgets import (QApplication, QMessageBox, QSystemTrayIcon)
 from osfoffline.application.main import OSFApp
 from osfoffline import utils
 
 
+def is_only_instance():
+    # Determine if there are more than the current instance of the application
+    # running at the current time.
+    return os.system("(( $(ps -ef | grep python | grep '[" +
+                     __main__.__file__[0] + "]" + __main__.__file__[1:] +
+                     "' | wc -l) > 1 ))") != 0
+
+
+def running_warning():
+    warn_app = QApplication(sys.argv)
+    QMessageBox.information(
+        None,
+        "Systray",
+        "OSF-Offline is already running. Check out the system tray."
+    )
+    warn_app.quit()
+    exit(0)
```

Comment:
Use `sys.exit` instead of `exit` here (may alleviate some issues in a threaded application)